provisioning/osc-install

```diff
@@ -85,14 +118,14 @@ setup_authentication() {
   yum -y install httpd-tools
   touch /etc/openshift-passwd
   for user in $users; do
-    htpasswd -b /etc/openshift-passwd ${user%:*} ${user#*:}
+    htpasswd -b /etc/openshift/openshift-passwd ${user%:*} ${user#*:}
   done
 
   # edit master.yml oauth settings
   sed -i -e 's/name: anypassword/name: apache_auth/' \
   -e 's/kind: AllowAllPasswordIdentityProvider/kind: HTPasswdPasswordIdentityProvider/' \
   -e '/kind: HTPasswdPasswordIdentityProvider/i \      file: \/etc\/openshift-passwd' \
```

Comment:
believe you need to update the "/etc/openshift-passwd" path here due to the change above on line 121(?)